cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/TransformExecutorFactory.java

```diff
@@ -48,6 +48,8 @@
   protected final PipelinePluginInstantiator pluginInstantiator;
   protected final Metrics metrics;
   protected Schema outputSchema;
+  protected boolean isMapPhase;
```

Comment:
It is set by `MapReduceTransformExecutorFactory` since it has access to hadoopContext